nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/graph-controls/operation-control/operation-control.component.html

```diff
@@ -99,14 +99,6 @@
                             (click)="copy(selection)">
                             <i class="fa fa-copy"></i>
                         </button>
-                        <button
-                            mat-icon-button
-                            class="primary-icon-button mr-2"
-                            type="button"
-                            [disabled]="!canPaste()"
-                            (click)="paste()">
```

Comment:
We should be able to also remove `canPaste` and `paste` from the underlying component.